integration-core/pom.xml

```diff
@@ -31,6 +31,11 @@
         </dependency>
 
         <dependency>
+            <groupId>com.symphony</groupId>
+            <artifactId>crypto</artifactId>
```

Comment:
Once this dependency is not open-sourced it will break Travis CI build. Then, our PR checker will be unavailable.

I think we need to discuss with all team members about this. I mean we need to think another approach to make sure our PR's will be checked.

Reply:
I've asked @mgavrilov-symphony about this on the Security Team Room.

Reply:
I saw in the output of the build that it complains about the missing 'version' parameter. The latest safe version for your usage is 1.48.2.

Reply:
Done.